tools/topology/sof-smart-amplifier.m4

```diff
@@ -122,7 +125,7 @@ dnl     time_domain, sched_comp)
 # Set 1000us deadline on core 0 with priority 0
 PIPELINE_PCM_ADD(sof/pipe-smart-amplifier-playback.m4,
 	SMART_PB_PPL_ID, SMART_PCM_ID, SMART_PB_CH_NUM, s32le,
-	1000, 0, SMART_AMP_CORE,
+	SMART_AMP_PERIOD, 0, SMART_AMP_CORE,
```

Comment:
@fuyuntsuo sorry, not following the changes to SMART_AMP here and how it relates ?

Reply:
The long process time (aprox. 8ms) and frame size (16ms in 16k sample rate) of igo_nr starve the scheduler and results in SMART_AMP underflow, ending up with smart_amp component reset and close. So we have to increase the buffer size of SMART_AMP.

Reply:
Got you, probably worth mentioning this in the comments.

Reply:
sure. will put more description next to SMART_AMP_PERIOD define.